packages/expo-image/android/src/main/java/expo/modules/image/events/ImageLoadStartEvent.java

```diff
@@ -0,0 +1,23 @@
+package expo.modules.image.events;
+
+import com.facebook.react.bridge.Arguments;
+import com.facebook.react.uimanager.events.Event;
+import com.facebook.react.uimanager.events.RCTEventEmitter;
+
+public class ImageLoadStartEvent extends Event<ImageLoadStartEvent> {
+  public static final String EVENT_NAME = "topLoadStart";
```

Comment:
Why is this called `topLoadStart` ?

Reply:
Oops, afair that was the convention some time ago, I didn't know React now handles `on<EventName>` too. Fixing!